src/main/java/com/dreammaster/gthandler/recipes/LaserEngraverRecipes.java

```diff
@@ -202,6 +202,16 @@ public void run() {
                     500 * 10,
                     983_040,
                     true);
+        
+            GT_Values.RA.addLaserEngraverRecipe(
+                    new ItemStack[] { GT_OreDictUnificator.get(OrePrefixes.lens, Materials.Diamond, 1L).copy().splitStack(0), 
```

Comment:
GT_OreDictUnificator.get(OrePrefixes.lens, Materials.Diamond, 0L) instead

Reply:
i guess with splitStack it is 0 but other laser engraver recipes was added different. Not sure what is a good coding style here ?